spring-cloud-kubernetes-discovery/src/main/java/org/springframework/cloud/kubernetes/discovery/KubernetesDiscoveryClientAutoConfiguration.java

```diff
@@ -11,6 +12,8 @@
 import org.springframework.context.annotation.Primary;
 
 @Configuration
+@EnableConfigurationProperties(KubernetesDiscoveryProperties.class)
```

Comment:
No need for `(KubernetesDiscoveryProperties.class)` since it's declared as a bean below.